src/main/java/com/ays/backend/user/service/AuthService.java

```diff
@@ -16,5 +18,11 @@ public interface AuthService {
      */
     UserDTO register(AdminRegisterRequest registerRequest);
 
-
+    /**
+     * Login to platform.
+     *
+     * @param loginRequest the loginRequest entity
+     * @return AuthResponse
+     */
+    AuthResponse login(AdminLoginRequest loginRequest);
```

Comment:
The service should return a domain object, while the controller should return a response object.

Reply:
I tried to use a dto named UserTokenDTO for this purpose in AuthService instead of returning a response object.